hack/update-codegen.sh

```diff
@@ -36,7 +36,7 @@ git clone git@github.com:kubernetes/code-generator.git
 popd
 pushd ${WORK_DIR}/go/src/k8s.io/code-generator
 # The version needs to match the one in Gopkg.toml
-git checkout kubernetes-1.15.0-alpha.0
+git checkout kubernetes-1.18.6
```

Comment:
I had to manually `cp -r github.com/kubernetes-sigs/kube-storage-version-migrator/pkg/clients pkg`